bulwark/checks.py

```diff
@@ -408,6 +408,35 @@ def is_same_as(df, df_to_compare, **kwargs):
     return df
 
 
+def matches_regex(df, pattern, columns=None, **kwargs):
```

Comment:
What do you think of passing a dictionary of column: pattern, so that different columns can be checked for different patterns in the same check? Otherwise you might have to call this check a number of times.